src/search.cpp

```diff
@@ -255,31 +255,22 @@ void MainThread::search() {
       &&  rootMoves[0].pv[0] != MOVE_NONE)
   {
       std::map<Move, int64_t> votes;
-      Value minScore = this->rootMoves[0].score;
-
-      // Find out minimum score and reset votes for moves which can be voted
-      for (Thread* th: Threads)
-      {
-          minScore = std::min(minScore, th->rootMoves[0].score);
-          votes[th->rootMoves[0].pv[0]] = 0;
-      }
 
       // Vote according to score and depth
-      auto square = [](int64_t x) { return x * x; };
       for (Thread* th : Threads)
-          votes[th->rootMoves[0].pv[0]] += 200 + (square(th->rootMoves[0].score - minScore + 1)
-                                                  * int64_t(th->completedDepth));
+      {
+          int64_t s = th->rootMoves[0].score + VALUE_INFINITE + 1; // Ensure positive
```

Comment:
This changes the voting scheme, I don't see that this would be equivalent...

Reply:
Why? It seems to me that if vote(th1) > vote(th2) before the change then it is still true after the change. Did I miss something?

Reply:
the precise value of votes is important now. Two threads that vote for a move with 10cp advantage is better than one vote with 13cp advantage.

Reply:
Ok I see. Thanks. I will revert the patch.